### DISCUSSION TITLE
"Only show favorites" on "/home"

### DISCUSSION THREAD

**Discussion by Michi0606** [music-assistant/support#1502] [Feature requests and ideas]
This is a feature request to have an option to show favorites only on home

**Comment by OzGav** [comment]
Sorry I don’t understand this?

**Reply by Michi0606** [reply]
Hello,
i am talking about the little heart icon which is shown for example on view "/albums" (Album View). if you click on that little heart then only your favorite albums will be shown, but on start page "/home" (with the little house icon) there is no heart icon, so all albums (and tracks, and playlists and so on) of all music providers will be shown there. in my plex account there is a music collection of someone else linked, and i dont want to see "random" music but my favorites only on start page. 

best regards Michael

**Reply by OzGav** [reply]
Ok that’s not really the vision for the main screen but let’s see if it gets any traction from others. Perhaps you could also see if you can split those two Plex collections.

**Comment by OzGav** [comment]
Oversome by UI improvements
